cli/src/main/java/com/devonfw/tools/ide/tool/vscode/Vscode.java

```diff
@@ -28,4 +32,14 @@ protected String getBinaryName() {
     return "code";
   }
 
+  @Override
+  public void installPlugin(PluginDescriptor plugin) {
```

Comment:
Also the active plugin installation is implemented differently:
In devonfw-ide we collect all active and all inactive plugins and then do one call of VSCode (`code`) for each list.
You trigger an invocation of VScode for each active plugin.
Assuming you have 10 active plugins configured this will launch VScode 10 times instead of only once what makes quite some difference in performance.

Reply:
I have changed the implementation completely. I would like to know if this is the correct way to go about this issue, so that I can keep working on the tests, because the tests are still missing.

Reply:
Thanks for your update. See my new review. This is now resolved.